pom.xml

```diff
@@ -23,7 +23,7 @@
     <groupId>org.cyclonedx</groupId>
     <artifactId>cyclonedx-core-java</artifactId>
     <packaging>jar</packaging>
-    <version>7.0.1-SNAPSHOT</version>
+    <version>8.0.0</version>
```

Comment:
as Steve said I think it's okay to include those changes in 7.0.1

Reply:
Done!